jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -115,81 +124,92 @@
             jdiAssert(element.get(name).isDisplayed() ? name + "displayed" : "hidden", Matchers.is(name + "displayed"));
         return this;
     }
-
+    
     /**
      * Check that at least one element is displayed
+     *
      * @return DataListAssert
      */
     @JDIAction("Assert that '{name}' has at least one displayed element")
     public DataListAssert<T, D> displayed() {
         jdiAssert(element.isDisplayed() ? "displayed" : "hidden", Matchers.is("displayed"));
         return this;
     }
-
+    
     /**
      * Check that all elements are hidden
+     *
      * @return DataListAssert
      */
     @JDIAction("Assert that '{name}' is hidden")
-    public DataListAssert<T, D> hidden() {
-        jdiAssert(map(element, this::isDisplayed), everyItem(Matchers.is(false)));
+    public DataListAssert<T, D> hidden(String ... messages) {
+        jdiAssert(map(element, this::isDisplayed), everyItem(Matchers.is(false)), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)